AirSkyBoat/AirSkyBoat
# Draft: Pulling the strings

**_I affirm:_**

## What does this pull request do?



## Steps to test these changes



Incomplete implementation of the ENM: Pulling the Strings.  Leaving this here in case anybody wants to pick it up or use as reference.

Things to be done:
   Fantoccini - Add in all mob skills for every weapon that's missing
   Fantoccini - Add in job abilities (seems like every job has 1 unique ability that it can only use once unless reset)
   Fantoccini - Randomize pets (different summons, puppets, etc.)
   Fantocciniman - Finish moblin special dice to force Fantoccini to use 2hr ability
   Fantocciniman - Add speech to skills
   ENM - Check mob Immunities
   ENM - Treasure
   ENM - Balance and testing
